src/components/Boards/Boards.tsx

```diff
@@ -1,22 +1,29 @@
 import React from 'react';
-import useGroups from '../../lib/hooks/useGroups';
-
-export default function Boards() {
-  const { data, isLoading, error } = useGroups();
-
-  if (isLoading) {
-    return <div>Loading...</div>;
-  }
-
-  if (error) {
-    return <div>Error Loading Boards</div>;
-  }
+import Paper from '@material-ui/core/Paper';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListAltIcon from '@material-ui/icons/ListAlt';
 
+export default function(props: {
+  boards: {
+    id: string;
+    name: string;
+  }[];
+}) {
   return (
-    <div>
-      {data.map(({ id, type, name }) => (
-        <div key={id}>{`${id}:${type}:${name}`}</div>
-      ))}
-    </div>
+    <Paper>
+      <List component="nav" aria-label="main mailbox folders">
+        {props.boards.map(({ id, name }) => (
+          <ListItem key={id} button={true}>
```

Comment:
Since `button` is a boolean prop, we can omit `true` here?

Reply:
I'm weird and prefer the verbose true. If you feel strongly, I can change it.

Reply:
Gotcha, we can keep it here